contributors/guide/help-wanted.md

```diff
@@ -21,41 +22,36 @@ grow and improve our community.
 
 ## Help Wanted
 
-Items marked with the `help wanted` label need to ensure that they are:
+Items marked with the `help wanted` label need to ensure that they meet these criteria:
 
 - **Low Barrier to Entry**
-
-  It should be tractable for new contributors. Documentation on how that type of
+  It should be easy for new contributors. Documentation on how that type of
   change should be made should already exist.
 
 - **Clear Task**
-
   The task is agreed upon and does not require further discussions in the
   community. Call out if that area of code is untested and requires new
   fixtures.
 
-  API / CLI behavior is decided and included in the OP issue, for example: _"The
-  new command syntax is `svcat unbind NAME [--orphan] [--timeout 5m]`"_, with
+  API and CLI behavior should be decided and included in the OP issue, for example: "The
+  new command syntax is `svcat unbind NAME [--orphan] [--timeout 5m]`", with
   expected validations called out.
 
-- **Goldilocks priority**
-
-  Not too high that a core contributor should do it, but not too low that it
-  isn't useful enough for a core contributor to spend time to review it, answer
-  questions, help get it into a release, etc.
+- **[Goldilocks](https://www.gutenberg.org/files/49001/49001-h/49001-h.htm#Page_207) priority**
```

Comment:
nit, the hyperlinked story is good and fun to read but wondering if we really want to add a link and possible risk of a broken link in the future? Also, it's story. The content in the section describe what we mean by `Goldilocks`.